.azure-pipelines-templates/release.yml

```diff
@@ -31,8 +31,10 @@ jobs:
     - script: |
         python3.7 -m venv env
         source ./env/bin/activate
-        python setup.py sdist
-        pip install twine
+        pip install -r requirements.txt
+        pip freeze > requirements.txt
```

Comment:
I'm not convinced this is a good idea. For one thing, someone who tries to use ccf in a script that requires a slightly newer psutil is going to have a very unpleasant time.

Can we instead set minimum versions manually?